tools/server/webui/src/components/ChatMessage.tsx

```diff
@@ -55,32 +49,6 @@ export default function ChatMessage({
   const nextSibling = siblingLeafNodeIds[siblingCurrIdx + 1];
   const prevSibling = siblingLeafNodeIds[siblingCurrIdx - 1];
 
-  // for reasoning model, we split the message into content and thought
-  // TODO: implement this as remark/rehype plugin in the future
-  const { content, thought, isThinking }: SplitMessage = useMemo(() => {
-    if (msg.content === null || msg.role !== 'assistant') {
-      return { content: msg.content };
-    }
-    let actualContent = '';
-    let thought = '';
-    let isThinking = false;
-    let thinkSplit = msg.content.split('<think>', 2);
-    actualContent += thinkSplit[0];
```

Comment:
If user doesn't explicitly enable `deepseek` reasoning format (which is the default behavior), this code is there to make sure the thinking is always parsed.

Unless we can control thinking format per-request, I don't think we can remove this code block

Reply:
reverted UI changes